packages/react-composites/tests/app/chat/LiveTestApp.tsx

```diff
@@ -30,7 +30,7 @@ export const LiveTestApp = (): JSX.Element => {
   const useFrLocale = Boolean(params.useFrLocale);
   const customDataModel = params.customDataModel;
   const useFileSharing = Boolean(params.useFileSharing);
-  const failFileDownload = Boolean(params.failDownload);
+  const failAttachmentDownload = Boolean(params.failDownload);
   const uploadedFiles = React.useMemo(() => (params.uploadedFiles ? JSON.parse(params.uploadedFiles) : []), []);
```

Comment:
Is uploadedFiles excluded from this rename?

Reply:
yes, viktoriia has addressed naming for download since it has been refracted and all APIs updated.

she left upload related API as is since it hasn't been refactored, we have another PR for it